content/pages/sponsorship.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Sponsorship"
+template: "page"
+---
+With your help, we can create the most accessible and fun space possible to learn, share, and grow digital accessibility knowledge in our community, through our [mission and values](/pages/mission).
+
+Since its inception, A11yTalks has hosted a variety of live talks and panels covering topics related to digital accessibility for beginners, developers, and creatives. All of these videos are streamed and then archived on the [A11yTalks YouTube channel](https://www.youtube.com/c/accessibilitytalks) with cleaned up captions.
+
+Our content is available to the public as a free resource for digital accessibility education as well as continuing education credits for certified International Association of Accessibility Professionals (IAAP).
+In 2022, we assembled an [all-volunteer team](https://a11ytalks.com/pages/about), passionately contributing time and energy to sustain the project, and we became fiscally hosted by a non-profit, Open Collective Foundation (501)(c)(3).
+
+Now, we need your support to help us meet our accessibility goals, compensate our amazing speakers, and improve our live stream productions to provide the best value possible for our community.
+
+## Budget Priorities for 2023
+
+1. __Live captions__ ($150/month, $1,800/annualy): Staying true to our value of accessibility, high quality live captions are our top priority to ensure our live stream is accessible to as many people as possible. Our current vendor provides live human-generated captions.
```

Comment:
- This needs to be a table so we can show how everything adds up to our monthly and annual budget.
- The details for each can be listed below the table.
- The item label in the table can jump link to the details.